tests/ops/models/test_application_config.py

```diff
@@ -221,7 +221,7 @@ def test_update_config_set(self, db):
         ApplicationConfig.update_config_set(db, CONFIG)
         # check a few specific config properties of different types on the database record
         db_config = ApplicationConfig.get_config_set(db)
-        assert dumps(db_config) == CONFIG.json()
+        assert dumps(db_config, separators=(",", ":")) == CONFIG.model_dump_json()
```

Comment:
`model_dump_json` stores in a more condensed format, to save space, so I'm adjusting the test